### ISSUE TITLE
New idea

### ISSUE THREAD

**Issue by royal-feng** [drmaj/UnDeepVO#2]
I wander a new idea about the pose_net . Because the input of the pose_net is 2 continuous frame without the estimate_depth ,it is precondition. If I have the Groundtruth of pose, May I train the pose_net with 2 continuous frame and its pose-GT? Can the pose_net have the ability to learn the pose between the 2 image in a supervised way? Just for test the pose_net, I understand the meaning of the unsupervised learning.

**Comment by ghost** [comment]
Yes, I'm sure that it would work. Similar work has been carried out before. Have a look at

- https://arxiv.org/abs/1702.01381
- https://arxiv.org/pdf/1505.07427.pdf

And the github repo:

- https://github.com/AaltoVision/relativeCameraPose
